obspy/imaging/waveform.py

```diff
@@ -1454,7 +1455,10 @@ def _add_zoomlevel_warning_text(self):
     def _remove_zoomlevel_warning_text(self):
         ax = self.fig.axes[0]
         if self._minmax_warning_text in ax.texts:
-            ax.texts.remove(self._minmax_warning_text)
+            if MATPLOTLIB_VERSION < [3, 6]:
+                ax.texts.remove(self._minmax_warning_text)
+            else:
+                self._minmax_warning_text.remove()
```

Comment:
`Text.remove` has always existed; there's no need to version-gate this.

Reply:
Thanks! Will remove the if/else tomorrow :rocket: